extensions/wikia/Wall/WallExternalController.class.php

```diff
@@ -163,13 +169,21 @@ public function votersListItems() {
 	}
 
 	public function switchWatch() {
+		try {
+			// SUS-664: Validate edit token
+			$this->checkWriteRequest();
+		} catch ( BadRequestException $e ) {
+			$this->setTokenMismatchError();
+			return false;
+		}
+
 		$this->response->setVal( 'status', false );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "status" 31 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)